### DISCUSSION TITLE
HIFI-GAN

### DISCUSSION THREAD

**Discussion by MaizeKai** [speechbrain/speechbrain#1719] [Q&A]
Hello, I use librosa to transfer the audio to mel spectrogram(n_mel=80, n_fft=1102, hop_length=275 sr=22050), and try to use the HIFI-GAN to reconstruct the audio, but the sound file made a very slight electric sound. Do you have any suggestions? Thank you

**Comment by BenoitWang** [comment]
Hi @MaizeKai , our model was trained with torchaudio mel-spectrograms and with a different setting (`n_fft=1024`, `hop_length=256`, etc.). However, in this case I would suggest to train a new model with your librosa mel-spectrograms and go for enough epochs.

**Reply by MaizeKai** [reply]
HI @BenoitWang , thank you for your suggestion, I will try to train a new model.

**Reply by wj-gxy** [reply]
Hello, I would like to know if the length of the output speech (wav) is consistent when you input a Mel spectrum into the trained model? Just like: wav (len=8219) ->mel (32, 80, 32) ->wav (len=?)
